public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -811,7 +832,7 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
             List<Long> analysisIds = new ArrayList<>();
             analysisIds.add(3300L);
             analysisIds.add(3301L);
-            List<AchillesAnalysis> ehrAnalysesList = achillesAnalysisDao.findAnalysisByIds(analysisIds, domainId);
+            List<AchillesAnalysis> ehrAnalysesList = achillesAnalysisDao.findAnalysisByIdsAndDomain(analysisIds, domainId);
```

Comment:
remove
`List<Long> analysisIds = new ArrayList<>();`
`analysisIds.add(3300L); `
`analysisIds.add(3301L); `
in favor of
```suggestion
            List<AchillesAnalysis> ehrAnalysesList = achillesAnalysisDao.findAnalysisByIdsAndDomain(ImmutableList.of(3300L, 3301L), domainId);
```

Reply:
Done thanks.